### DISCUSSION TITLE
v6 remove all regions

### DISCUSSION THREAD

**Discussion by g-no** [katspaugh/wavesurfer.js#2823] [Q&A] [answered]
In v6 how can I remove all regions?

**Comment by katspaugh** [comment, marked as answer]
There's a [clear](https://github.com/katspaugh/wavesurfer.js/blob/v6/src/plugin/regions/index.js#L233-L240) function, but tbh I don't remember how you're supposed to call it.

**Comment by g-no** [comment]
Thanks for the quick answer! I'll figure that out.

**Reply by g-no** [reply]
It's simply: `wavesurfer.regions.clear()`.
